_docs/tasks/installing-istio.md

```diff
@@ -37,14 +37,20 @@ default namespace. They can be modified for deployment in a different namespace.
 
 1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS.
 
-2. Extract the installation file, and change directory to the location where the files were extracted. Following instructions are relative to this installation directory.
+1. Extract the installation file, and change directory to the location where the files were extracted. Following instructions are relative to this installation directory.
    The installation directory contains:
     * yaml installation files for Kubernetes
     * sample apps
     * the `istioctl` client binary, needed to inject Envoy as a sidecar proxy, and useful for creating routing rules and policies.
-    * a text file containing the Istio version.
+    * the istio.VERSION configuration file.
 
-3. Run the following command to determine if your cluster has [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/)
+1. Add the `istioctl` client to your PATH. For example, run the following commands on a Mac system:
+
+   ```bash
+   sudo ln -s $PWD/istioctl /usr/local/bin/
```

Comment:
There are other ways to do it. For instance, i have "." in my PATH. 
bin is a good idea. If the installation archive ends up being different than the instructions, please update the instructions.